webpack.config.js

```diff
@@ -18,13 +18,18 @@ const privateKey = fs.readFileSync(path.join(__dirname, 'private.key'), 'utf8');
  * Generate a JWT token using the current requesting users ID, your project ID (pid) and segment ID (sid). Return
  * the generated token back to the client.
  */
-function serveJWT(req, res){
-    const {userID} = req.params;
-    if(!userID){
+function serveJWT(req, res) {
```

Comment:
Maybe worth setting up prettier configs for this repo.

Reply:
Added and had to update the `.eslintrc` file so that it would accommodate some of the prettier changes, hence a change to the `brace-style` eslint rule.